### ISSUE TITLE
[Bug]: (strings) Non plain English, non translatable property name in queueUpdate() success message

### ISSUE THREAD

**Issue by silopolis** [nextcloud/server#37538]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] Nextcloud Server **is** running on 64bit capable CPU, PHP and OS.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

The queueUpdate() success message uses the share's raw property name, which is not that English users friendly and not translatable.

Plus, my shamy PHP non-skills make me believe that it doesn't return all properties when several properties are passed to queueUpdate().

https://github.com/nextcloud/server/blob/master/apps/files_sharing/src/mixins/SharesMixin.js#L280

### Steps to reproduce

Update share properties and get a success message

### Expected behavior

Have user messages in plain English and translatable, whether one or several properties are passed to queueUpdate().

### Installation method

None

### Nextcloud Server version

master

### Operating system

None

### PHP engine version

None

### Web server

None

### Database engine version

None

### Is this bug present after an update or on a fresh install?

None

### Are you using the Nextcloud Server Encryption module?

None

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### List of activated Apps

```shell
Files
```

**Comment by szaimen** [comment]
cc @Pytal

**Comment by Pytal** [comment]
This would need a mapping between the property name and readable string for all properties passed to `queueUpdate()` https://github.com/search?q=repo%3Anextcloud%2Fserver+queueUpdate&type=code

And yes since there may be multiple properties (introduced in https://github.com/nextcloud/server/pull/20938) we'd have to show each one

**Comment by joshtrichards** [comment]
Fixed in #48696
